docs/Conformance Criteria/README-Conformance Criteria V2.md

```diff
@@ -0,0 +1,349 @@
+   <th style="background-color:#AAAAAA"></th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th></th>
+   <td><b>No Zowe CLI plugin installation requirement: </b> If the extender makes use of a Zowe CLI profile other than the Zowe Explorer default `zosmf` then the extension must not make any assumptions that a matching Zowe CLI plugin has been installed in the Zowe Explorer user's environment.</td>
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">3</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th></th>
+   <td><b>Publication tag:</b> If the extension is published in a public catalog or marketplace such as Npmjs, Open-VSX, or VS Code Marketplace, it uses the tag or keyword "Zowe" so it can be found when searching for Zowe and be listed with other Zowe offerings.</td>
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">4</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th></th>
+   <td><b>Support:</b> Extension has documentation with instructions on how to report problems that are related to the extension and not Zowe Explorer. It needs to explain how users can determine if a problem is related to the extension or Zowe Explorer.</td>
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">5</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th></th>
+   <td><b>User settings consistency:</b> Extender provides a consistent user settings experience. For VS Code extensions, extender follows the recommended naming convention for configuration settings as described in VS Code's <a href="https://code.visualstudio.com/api/references/contribution-points#contributes.configuration">configuration contribution documentation</a></td>
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">6</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th></th>
+   <td><b>Using .zowe in user settings:</b> Extender avoids starting setting names with the prefix `zowe.`, which is reserved for Zowe Explorer and and other extensions maintained by the Zowe Project
+.</td>
+ </tr>
+
+ <tr>
+   <th style="background-color:#555555">7</th>
+   <th style="background-color:#555555">v2</th>
+   <th style="background-color:#AAAAAA"></th>
+   <th style="background-color:#AAAAAA">x</th>
+   <th></th>
+   <td><b>Error message consistency:</b> Extension follows the recommended error message format indicated in the <a href=https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/extenders/Error%20Handling.md>Zowe Explorer extensibility documentation</a> to provide a consistent user experience with Zowe Explorer.</td>
```

Comment:
Did we mean to change this Error Message Consistency item to be required for V2? I see it listed as a planned change in the earlier version of Zowe Explorer's Office Hours, but don't remember taking it off the list.